openedx_events/learning/signals.py

```diff
@@ -350,3 +353,65 @@
         "submission": ORASubmissionData,
     },
 )
+
+
+# .. event_type: org.openedx.learning.ora.submission.created.v1
+# .. event_name: ORA_SUBMISSION_CREATED
+# .. event_description: Emitted when a new ORA submission is created
+# .. event_data: ORASubmissionData
+# Warning: This event is currently incompatible with the event bus, list/dict cannot be serialized yet
+ORA_SUBMISSION_CREATED = OpenEdxPublicSignal(
```

Comment:
This is something I'm not expecting here.

Reply:
Mistake during rebasing. Deleted